frontend/src/components/tab/TabCurrentTab.tsx

```diff
@@ -0,0 +1,10 @@
+import { useTabs } from '@/hooks/useTabs';
+import * as S from './Tab.styled';
+
+export default function TabCurrentTab() {
+  const { selectedIndex, tabList } = useTabs();
+
+  return (
+    <S.TabCurrentContentContainer>{tabList[selectedIndex].content}</S.TabCurrentContentContainer>
```

Comment:
사소하지만, 여기 컴포넌트 명에 따라서 CurrentTabContainer로 바꿔도 좋을 것 같아요~!
```suggestion
    <S.CurrentTabContainer>{tabList[selectedIndex].content}</S.CurrentTabContainer>
```

Reply:
짚어주셔서 감사합니다 :)